sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -351,6 +351,11 @@ class CudfMin(override val dataType: DataType) extends CudfAggregate {
   override val name: String = "CudfMin"
 }
 
+/** Check if all values in a boolean column are trues */
```

Comment:
Same comment as for `CudfAny`. It would be good to have a comment explaining why we are using `CudfMin(BooleanType)`

Reply:
Updated.